### PR TITLE
mention ipn website

### PR DESCRIPTION
**Pull request by pbauer**

### CODE REVIEW

**Review by tisto — APPROVED** [review]
@pbauer awesome!!!

File: news/5241.documentation [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Add new website IPN
```
Comment (tisto) [inline news/5241.documentation:1]: @sneridagh don't we have an "internal" towncrier option?

Comment (stevepiercy) [reply news/5241.documentation:1]: Yes: https://6.docs.plone.org/contributing/index.html#contributing-change-log-label

Comment (pbauer) [reply news/5241.documentation:1]: yes but that's docs, not internal imho

**Pull request merged**